### PR TITLE
overall doc edits

### PR DESCRIPTION
**Pull request by rosemarymarano**
Reviewed entire operator documentation, fixed and homogenized wherever needed.

### CODE REVIEW

File: docs-source/content/faq/external-clients.md [hunk 1]
```diff
@@ -29,15 +29,15 @@ Here are the steps:
...
+The Kubernetes `NodePorts` approach for giving external WebLogic EJB or JMS clients access to a Kubernetes hosted WebLogic cluster involves configuring a network channel on the desired WebLogic cluster that accepts T3 protocol traffic, and deploying a Kubernetes `NodePort` that redirects external network traffic on the Kubernetes nodes to the network channel.
 
 Here are the steps:
 
 - Configure a custom channel for the T3 protocol in WebLogic that specifies an external address and port that are suitable for remote client use.  See [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
 
-- Deploy a Kubernetes NodePort to publicly expose the WebLogic ports. See [Setting up a NodePort](#setting-up-a-nodeport).
+- Deploy a Kubernetes `NodePort` to publicly expose the WebLogic ports. See [Setting up a `NodePort`](#setting-up-a-nodeport).
```
Comment (markxnelson) [inline docs-source/content/faq/external-clients.md:40]: I don't think that "deploy" makes sense here, "create" is probably more accurate, or "define" or "expose"

File: docs-source/content/faq/external-clients.md [hunk 2]
```diff
@@ -136,21 +136,21 @@ In this example:
 
 - Additional steps are required for external clients beyond configuring the custom channel - see [Approaches](#approaches).
 
-#### Setting up a NodePort
+#### Setting up a `NodePort`
 
 ##### Getting started
 
-A Kubernetes NodePort exposes a port on each machine that hosts the Kubernetes cluster where the port is accessible from outside of a Kubernetes cluster. This port redirects network traffic to pods within the Kubernetes cluster. Setting up a Kubernetes NodePort is one approach for giving external WebLogic clients access to JMS or EJBs.
+A Kubernetes `NodePort` exposes a port on each machine that hosts the Kubernetes cluster where the port is accessible from outside of a Kubernetes cluster. This port redirects network traffic to pods within the Kubernetes cluster. Setting up a Kubernetes `NodePort` is one approach for giving external WebLogic clients access to JMS or EJBs.
```
Comment (markxnelson) [inline docs-source/content/faq/external-clients.md:143]: each machine that hosts -> each worker node in

Comment (markxnelson) [reply docs-source/content/faq/external-clients.md:143]: (they are not exposed on masters)

File: docs-source/content/faq/external-clients.md [hunk 3]
```diff
@@ -136,21 +136,21 @@ In this example:
...
 
-If an EJB or JMS service is running on an Administration Server, then you can skip the rest of this section and use the `spec.adminServer.adminService.channels` domain resource attribute to have the operator create a NodePort for you. See [Reference - Domain resource]({{<relref "/reference/domain-resource/_index.md">}}). Otherwise, if the EJB or JMS service is running in a WebLogic cluster or standalone WebLogic Managed Server, and you desire to provide access to the service using a NodePort, then the NodePort must be deployed 'manually' - see the following sample and table.
+If an EJB or JMS service is running on an Administration Server, then you can skip the rest of this section and use the `spec.adminServer.adminService.channels` domain resource attribute to have the operator create a `NodePort` for you. See [Reference - Domain resource]({{<relref "/reference/domain-resource/_index.md">}}). Otherwise, if the EJB or JMS service is running in a WebLogic cluster or standalone WebLogic Managed Server, and you desire to provide access to the service using a `NodePort`, then the `NodePort` must be deployed 'manually' - see the following sample and table.
```
Comment (markxnelson) [inline docs-source/content/faq/external-clients.md:145]: deployed -> created/exposed/defined

File: docs-source/content/faq/external-clients.md [hunk 4]
```diff
@@ -136,21 +136,21 @@ In this example:
...
 {{% notice note %}}
-Setting up a NodePort usually also requires setting up a custom network channel. See [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
+Setting up a `NodePort` usually also requires setting up a custom network channel. See [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
 {{% /notice %}}
 
-##### Sample NodePort resource
+##### Sample `NodePort` resource
 
-The following NodePort YAML deploys an external node port of `30999` and internal port `7999` for a domain UID of `DOMAIN_UID`, a domain name of `DOMAIN_NAME`, and a cluster name of `CLUSTER_NAME`. It assumes that `7999` corresponds to a T3 protocol port of a channel that's configured on your WebLogic cluster.
+The following `NodePort` YAML deploys an external node port of `30999` and internal port `7999` for a domain UID of `DOMAIN_UID`, a domain name of `DOMAIN_NAME`, and a cluster name of `CLUSTER_NAME`. It assumes that `7999` corresponds to a T3 protocol port of a channel that's configured on your WebLogic cluster.
```
Comment (markxnelson) [inline docs-source/content/faq/external-clients.md:153]: deploys ..

File: docs-source/content/faq/external-clients.md [hunk 5]
```diff
@@ -29,15 +29,15 @@ Here are the steps:
 
 - Review the [Security notes](#security-notes).
 
-##### Kubernetes NodePorts
+##### Kubernetes `NodePorts`
 
-The Kubernetes NodePorts approach for giving external WebLogic EJB or JMS clients access to a Kubernetes hosted WebLogic cluster involves configuring a network channel on the desired WebLogic cluster that accepts T3 protocol traffic, and deploying a Kubernetes NodePort that redirects external network traffic on the Kubernetes nodes to the network channel.
+The Kubernetes `NodePorts` approach for giving external WebLogic EJB or JMS clients access to a Kubernetes hosted WebLogic cluster involves configuring a network channel on the desired WebLogic cluster that accepts T3 protocol traffic, and deploying a Kubernetes `NodePort` that redirects external network traffic on the Kubernetes nodes to the network channel.
```
Comment (markxnelson) [inline docs-source/content/faq/external-clients.md:35]: I would add a note:  The NodePort approach is only available when worker nodes are accessible by the clients, for example when they have public IP addresses.  If private worker nodes are used and access to them is only possible through a load balancer or bastion, then the NodePort approach is not a valid option to provide access to external clients.

File: docs-source/content/faq/namespace-management.md [hunk 6]
```diff
@@ -5,23 +5,23 @@ draft: false
 weight: 1
 ---
 
-Each WebLogic operator deployment manages a number of Kubernetes namespaces. For information about setting domain namespaces, see [Operator Helm configuration values]({{< relref "/userguide/managing-operators/using-the-operator/using-helm#operator-helm-configuration-values" >}}). A number of Kubernetes resources
+Each operator deployment manages a number of Kubernetes namespaces. For information about setting domain namespaces, see [Operator Helm configuration values]({{< relref "/userguide/managing-operators/using-the-operator/using-helm#operator-helm-configuration-values" >}}). A number of Kubernetes resources
```
Comment (markxnelson) [inline docs-source/content/faq/namespace-management.md:8]: "setting domain namespaces" feels awkward

Comment (markxnelson) [reply docs-source/content/faq/namespace-management.md:8]: probably should add a note to say that there can be multiple operators in a k8s cluster, and in that case, care must be taken to ensure their respective lists of domainNamespaces do not overlap.

File: docs-source/content/faq/namespace-management.md [hunk 7]
```diff
@@ -53,8 +53,8 @@ If you don't know the release name of the operator, you can use `helm ls` to lis
 $ helm ls
 ```
 
-#### Add a Kubernetes namespace to an operator
-If you want a WebLogic operator deployment to manage a namespace, you need to add the namespace to the operator's `domainNamespaces` list. Note that the namespace has to be pre-created, for example, using the `kubectl create` command.
+#### Add a Kubernetes namespace to the operator
+If you want an operator deployment to manage a namespace, you need to add the namespace to the operator's `domainNamespaces` list. Note that the namespace has to be pre-created, for example, using the `kubectl create` command.
```
Comment (markxnelson) [inline docs-source/content/faq/namespace-management.md:57]: to be pre-created -> already exist

File: docs-source/content/faq/oci-lb.md [hunk 8]
```diff
@@ -9,19 +9,19 @@ If you are running your Kubernetes cluster on Oracle Container Engine
...
 IP addresses only, which are not reachable from outside the cluster.  
-Instead, you can use an OCI Load Balancer to provide access
+Instead, you can use an OCI load balancer to provide access
 to services running in OKE.
 
 {{% notice note %}}
-It is also possible, if desirable, to have an OCI Load Balancer route
-traffic to an ingress controller running inside the Kubernetes cluster
-and have that ingress controller in turn route traffic to services in the
+It is also possible, if desirable, to have an OCI load balancer route
```
Comment (markxnelson) [inline docs-source/content/faq/oci-lb.md:22]: we need to update this -- our recommendation should be to use both an oci lb which sends traffic to the ingress controller and then handle routing on the ingress.  this is fine for now - but we should probably make a jira so we (i) don't forget to update this

File: docs-source/content/faq/oci-lb.md [hunk 9]
```diff
@@ -9,19 +9,19 @@ If you are running your Kubernetes cluster on Oracle Container Engine
...
+traffic to an Ingress controller running inside the Kubernetes cluster
```
Comment (markxnelson) [inline docs-source/content/faq/oci-lb.md:23]: I think "Ingress Controller" might be more correct for this

File: docs-source/content/userguide/experimental/istio.md [hunk 10]
```diff
@@ -6,7 +6,7 @@ weight: 1
 
 ### Overview
 
-WebLogic Kubernetes Operator version 2.3 includes experimental support for Istio 1.2.2.
+WebLogic Server Kubernetes Operator version 2.3 includes experimental support for Istio 1.2.2.
```
Comment (markxnelson) [inline docs-source/content/userguide/experimental/istio.md:9]: 2.3 and later
1.2.2 and later

File: docs-source/content/userguide/experimental/istio.md [hunk 11]
```diff
@@ -16,22 +16,22 @@ other components and services that have tracing enabled.
 
 ### Limitations
 
-The current experimental support for Istio has the current limitations:
+The current experimental support for Istio has these limitations:
 
-* It is tested only with Istio 1.2.2, however it is tested with both single and 
+* It is tested with Istio 1.2.2 only, however it is tested with both single and
```
Comment (markxnelson) [inline docs-source/content/userguide/experimental/istio.md:21]: only -> and later (up to 1.5)

File: docs-source/content/userguide/introduction/introduction.md [hunk 12]
```diff
@@ -53,15 +53,15 @@ Container Services for use with Kubernetes* on OCI Compute, and on "authorized c
...
 
 AKE support and limitations:
 
-* Both "domain in image" and "domain on persistent volume" models are supported.  
-* For domain on persistent volume we support Azure Files volumes accessed through
+* Both Domain in Image and Domain in PV domain home source types are supported.  
+* For Domain in PV, we support Azure Files volumes accessed through
   a persistent volume claim - see [here](https://docs.microsoft.com/en-us/azure/aks/azure-files-volume).
-* Azure Load Balancers are supported when provisioned using a Kubernetes service of `type=LoadBalancer`.
+* Azure load balancers are supported when provisioned using a Kubernetes service of `type=LoadBalancer`.
```
Comment (markxnelson) [inline docs-source/content/userguide/introduction/introduction.md:64]: I think Azure Load Balancer is a product name

File: docs-source/content/faq/external-clients.md (hunk 2 above)
Comment (rosemarymarano) [reply docs-source/content/faq/external-clients.md:143]: Do you want this parenthetical comment, (they are not exposed on masters), added also?

Comment (markxnelson) [reply docs-source/content/faq/external-clients.md:143]: i think it is subject to configuration... either leave it out, or say `..not *normally* exposed..`

File: docs-source/content/faq/external-clients.md (hunk 1 above)
Comment (rjeberhard) [reply docs-source/content/faq/external-clients.md:40]: I like "expose".  Kubernetes documentation uses this term: https://kubernetes.io/docs/concepts/services-networking/service/

File: docs-source/content/faq/oci-lb.md (hunk 9 above)
Comment (rosemarymarano) [reply docs-source/content/faq/oci-lb.md:23]: Hmmm. We don't capitalize controller anywhere else in the docs, except where it is part of a product name, such as Voyager Ingress Controller. In the Kubernetes docs, it's not capitalized (and Ingress is not consistently capitalized! Sheesh). Still want me to capitalize these two instances of controller or all of them?

Comment (markxnelson) [reply docs-source/content/faq/oci-lb.md:23]: ok that's fine, leave it as is

File: docs-source/content/faq/external-clients.md (hunk 1 above)
Comment (rosemarymarano) [reply docs-source/content/faq/external-clients.md:40]: I changed all the instances of "deploy" to "expose", except in this one bulleted item, where I thought that "define" was the correct term to use, "Define a Kubernetes `NodePort` to publicly expose the WebLogic ports. Please correct me if I'm wrong and I will change it to "expose."

Comment (rjeberhard) [reply docs-source/content/faq/external-clients.md:40]: That sounds reasonable to me.

**Pull request merged**